identity/manager.go

```diff
@@ -68,3 +71,8 @@ func (idm *identityManager) HasIdentity(identityString string) bool {
 
 	return err == nil
 }
+
+func (idm *identityManager) Register(id dto.Identity) (err error) {
```

Comment:
Dont mix RestClient into identity manager, it started dogin 2 responsibilieties

Reply:
What if registration will be doing more than just calling our api endpoint? for example broadcasting event? identity manager as a facade service for all identity management functionality seems to be the right plays to keep all related code in one place. Doing 2 or more responsibilities should viewed as providing some kind of services to caller like register , create, find etc.
If we use low level actions (call mysterium api) directly, refactoring will be problematic later as caller will need to know more and more implementation details about registering new identity.

Reply:
@Waldz This is more a question to you :)

Reply:
But it's strange&inefficient to load `MysteriuRestClient`, for action like `GetIdentity`
Later I see responsibilieties like this:
`IdentityManager` - jungle with local identities
`IdentityServiceClient` - jungle with identity endpoint